x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/rollup/v2/RollupActionSingleNodeTests.java

```diff
@@ -392,5 +462,49 @@ private CompositeAggregationBuilder buildCompositeAggs(String name, RollupAction
     public interface SourceSupplier {
         XContentBuilder get() throws IOException;
     }
+
+    private String createDataStream(boolean hidden) throws Exception {
```

Comment:
is there a specific reason why `createDataStream` takes in `hidden`? I do not see it ever something other than `false`.

Reply:
This was a left over from the copying the method from the data streams test. I don't see a reason for testing rollups on hidden data streams, so I will remove it. Thanks for spotting this.